spi/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@ if (NOT PICO_NO_HARDWARE)
     add_subdirectory(mpu9250_spi)
     add_subdirectory(bme280_spi)
     add_subdirectory(spi_dma)
+    add_subdirectory(eink_mono_spi)
```

Comment:
Alphabetical sorting :wink:

Reply:
Left this one at the end until the other PR gets merged to prevent conflicts :D